docs/v1/P4Runtime-Spec.mdk

```diff
@@ -4042,9 +4043,9 @@ the P4Info, the opaque target-dependent forwarding-pipeline configuration data
 to uniquely identify such configuration. See the [Forwarding-Pipeline
 Configuration](#sec-p4-fwd-pipe-config) section for details.
 
-A P4Runtime server running on an a typical device may not support
+A P4Runtime server running on an a non-programmable device may not support
```

Comment:
there is a typo that is not of your making on this line, there is an extra article ("an a")

Reply:
Done.